### ISSUE TITLE
[TestCase] Recurrent Network Test Case

### ISSUE THREAD

**Issue by myungjoo** [nnsuite/nnstreamer#738]
- [x] Generate a test case file of video/x-raw, "SEQ256", 4x4 BGRx with 256 frames, where the pixel values are enumerated from 0 to 255 for (ALLCOLOR,0,0) and from 255 to 0 for (ALLCOLOR,3, 3), and 0 to 255 for (RED, 1, 1).
    - It is asymmetric to check if there is any flips or shifts
    - #754
- [x] Create a custom filter, "LSTM1", that takes two tensors of 4:4:4 (C:W:H), Ta and Tb, and returns Ta for (\*:0:\*), Tb for (\*:3:\*), (Ta+Tb)/2 for (\*:1,2:\*)
- [x] Create a pipeline of: 
```
filesrc(SEQ256) --> tensor_converter -Ta-+--> tensor_mux --> LSTM1 --> tee --+--> queue --> filesink(RESULT)
                                         +<--Tb------------ queue <----------+
```
- [x] Add a synchronization policy for ```tensor_mux``` that supports this scenario
    - A specified sinkpad, B, will generate a dummy tensor for the first frame
    - Synchronization is driven by a specified sinkpad, A. (A != B)
- [x] Check if RESULT is as desired. (Generate golden test case)
    - The queue for Tb is going to be PITA for exact clocked synchronization!

**Comment by myungjoo** [comment]
CC: @jijoongmoon (he has been already doing the 4th/5th subissue)

**Comment by myungjoo** [comment]
Do This After #504 and #871. Postpone it to 1812B

**Comment by jijoongmoon** [comment]
- Tensor_reposink, Tensor_reposrc are added for this issue.
- Add nnstreamer_repo testcases to check
- Add nnstreamer_repo_rnn exmaple.
- Add nnstreamer_repo_lstm example.
